pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
     "botocore==1.35.96",
     "distro==1.9.0",
     "django==4.2.16",
+    "kubernetes>=33.1.0",
```

Comment:
ok, I think these are all dev deps, rou main prod deps are in https://github.com/ansible/metrics-utility/blob/devel/setup.cfg#L13

and then we'll need to talk to PDE to to add this dependency to metrics-utility RPM, which then will get to built image so the operator can use it. (afaik we still built from RPMs, I haven't played with Konflux yet)